ESPWebThingAdapter.h

```diff
@@ -167,6 +167,33 @@ class WebThingAdapter {
         prop["type"] = "string";
         break;
       }
+
+      prop["readOnly"] = property->readOnly;      
+      
+      if (property->unit != "")
+      {
```

Comment:
Brace style should match with other places in the file, (brace on same line as statement)